### PR TITLE
Dev -> Prod

### PR DESCRIPTION
**Pull request by jkonrath-postman**
markeng-2880 updated Auto Flex policy copy #5169
remove signup newsletter section #5171
TW-1393 Update "entities" to "elements" in LC #5165
TW-1155 Refresh mocking with examples #5173
TW-683 Updates to run collection in CI/CD section #5179
TW-1412 doc sunsetting old app versions #5167
Fixed anchors in docs #5176
TW-1423 sending the first request refresh #5175
TW-1376 update slack integrations #5180

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 1]
```diff
@@ -22,32 +22,37 @@ contextual_links:
...
 ---
 
-The Postman to Slack integration enables you to send Postman notifications to a Slack channel. You can send the results of a Postman monitor run, notifications received in the Postman notification center, or activity in your Team Activity Feed.
+The Postman to Slack integration enables you to send Postman notifications to a Slack channel. You can send the results of a Postman monitor run, notifications received in the Postman notification center, or activity in your team activity feed.
 
-## Configuring Postman with Slack
+## Contents
+
+* [Configure Postman with Slack](#configure-postman-with-slack)
+* [Post monitoring results](#post-monitoring-results)
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/slack.md:30]: Contents is missing an item after Post monitoring results:
`[Receive Postman notifications](#receive-postman-notifications)`

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/integrations/available-integrations/slack.md (hunk 1 above)
Comment (petewitcher-postman) [reply src/pages/docs/integrations/available-integrations/slack.md:30]: Thanks Mark, I've created a PR to fix this:
https://github.com/postmanlabs/postman-docs/pull/5182

Comment (petewitcher-postman) [reply src/pages/docs/integrations/available-integrations/slack.md:30]: This is fixed.

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

**Pull request merged**